src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -173,8 +146,42 @@ export default class SimidMapCreative extends BaseSimidCreative {
     const adContainer = document.getElementById('adContainer');
     adContainer.appendChild(returnToAdButton);
     adContainer.appendChild(skipAdButton);
+    this.createTravelDisplay_();
+    this.googleMapsClient_.setTravelMethodElement(document.getElementById("travel_method"));
+    this.googleMapsClient_.setTimeDisplayElement(document.getElementById("time_display"));
+  }
```

Comment:
If you call createMapState_ before creating the googleMapsClient object, can you pass these elements into the constructor and remove the setters?

Reply:
good idea, made that change